other/plague/plague bearer

```diff
@@ -2,14 +2,14 @@
 __Basics__ 
 The Plague Bearer's disease is lethal and highly contagious. Their desire is to kill off all other players.
 __Details__
-They start off with rodents that carry the plague. They have one rodent for every five players in the game (rounded).
-If there are several Plague Bearers they know each other. The Plague Bearers rodent counts are combined and each Plague Bearer may use one rodent per night. 
-During every night, they may free one of their rodents to contaminate another player. Contaminated players will be affected by the disease.
+The Plague Bearer starts the game with rodents that carry the plague. They have one rodent for every five players in the game (rounded).
+If there are several Plague Bearers then they know each other, and their rodent counts are combined. Each Plague Bearer may use one rodent per night. 
+Each night, the Plague Bearer may free one of their rodents to contaminate another player. Contaminated players will be affected by the disease. The Plague Bearer cannot be contaminated.
 If the Plague Bearer attempts to contaminate an already contaminated player, they are informed, but the rodent is still used up.
 Contaminating a player is an immediate ability. Contaminated players are contagious, and remain so till their death.
-Any player who visits or is visited by a contagious player is also immediately contaminated. Any type of action (target, attack, investigation, etc) is counted as a visit.
+Any player who visits or is visited by a contagious player is immediately contaminated. Any type of action (target, attack, investigation, etc) is counted as a visit.
 At the start of the second night after being contaminated the player is informed that they are contaminated.
-At the end of the second night after being contaminated the player will stop being contagious and die (not an attack).
-If no members of the plague team remain all players are no longer contaminated, however they are not informed about this.
-If the Plague Bearer attempts to use a rodent on the Plague Doctor, the attempt fails, the Plague Doctor immediately dies (not an attack) and the rodent is *not* used up.
+At the end of the second night after being contaminated the player will cease being contagious and die. This is not an attack.
```

Comment:
don't think 'cease' sounds good here

Reply:
I agree

Reply:
changed to stop, sounds much better

Reply:
That was the original wording anyway, not sure why @cryptidjayden changed it.